shared/actions/json/settings.json

```diff
@@ -174,9 +174,6 @@
     "processorProfile": {
       "durationSeconds": "number"
     },
-    "waitingForResponse": {
```

Comment:
using the waiting store instead which also lets us remove a bunch of chainGenerators